biz_portal/apps/portal/tests/test_admin.py

```diff
@@ -192,17 +219,15 @@ def test_muni_admin_edit_non_muni_businesses_denied(self):
         business = models.Business.objects.get(pk=2)
         self.assertNotEqual(business.supplied_name, "DEADBEEF")
 
-        view_response = self.client.get(
```

Comment:
why did you remove this? This is an important check that you can't edit a muni you're not an admin of.

Reply:
Because inline directors broke the assertion

Reply:
Rules were wrong. I fixed the problem.

https://github.com/OpenUpSA/biz-portal/pull/57/commits/a9cc2b2e2a757e338f71b611f8b491e3822c047d